tests/jballerina-unit-test/src/test/resources/test-src/statements/comment/comments.bal

```diff
@@ -45,7 +45,7 @@ final Day TUESDAY = "TUESDAY"; // enumerator Tuesday
 
 
 @http:ServiceConfig {basePath:"/FooService"} // http config annotation
-service FooService on new http:Listener(9093) {
+service FooService on new http:Listener(12909) {
```

Comment:
Can't we use the `http:MockListener` instead?

Reply:
Interesting, `http:MockListener` is new to me, will try that out.
Thank you :)